src/commands/analytics/log-flow.ts

```diff
@@ -116,7 +116,7 @@ export default class ShowLogFlowCommand extends AppCommand {
 
     // adding properties
     if (logEntry.properties != null) {
-      const logProperties: { [propertyName: string]: string } = JSON.parse(logEntry.properties);
+      const logProperties: { [propertyName: string]: string } = JSON.parse(logEntry.properties.propertyName);
```

Comment:
Hey guys could you please take a look if this change is correct or I'm missing something?

Reply:
@yimysty Could you take a look at this and verify the change?

Reply:
sure!

Reply:
![image](https://user-images.githubusercontent.com/8811182/32290732-32580976-bef8-11e7-88bb-447a6eeff20a.png)
the response is like this.. not quite get this part, seems there not `propertyName` property under `properties`?

Reply:
Is that you saying the change is good, or that you'll look at it?

Reply:
I will look at it.